idaes/models/properties/modular_properties/base/generic_property.py

```diff
@@ -3715,33 +3857,121 @@ def rule_surf_tens_phase(b, p):
             raise
 
     def _therm_cond_phase(self):
+        units = self.params.get_metadata().derived_units
         try:
+            self.therm_cond_phase = Var(
+                self.phase_list,
+                doc="Thermal conductivity for each phase",
+                units=units["thermal_conductivity"],
+                initialize=pyunits.convert_value(
+                    1,
+                    from_units=pyunits.W / pyunits.m / pyunits.K,
+                    to_units=units["thermal_conductivity"],
+                ),
+            )
 
             def rule_therm_cond_phase(b, p):
-                return get_phase_method(b, "therm_cond_phase", p)(b, p)
+                try:
+                    return b.therm_cond_phase[p] == get_phase_method(
+                        b, "therm_cond_phase", p
+                    )(b, p)
+                except GenericPropertyPackageError:
+                    # What's the best way to communicate to the user that therm_cond_phase isn't meaningful here?
```

Comment:
I am not sure there is a good way - there comes a point where we have to rely on user-knowledge, and if the user lacks the knowledge the issue is more "why are you using IDAES" than with specific subparts of the tool itself.